setup_google_colab.py

```diff
@@ -9,24 +9,15 @@ def download_github_code(path):
 
 
 def setup_common():
-    os.system("pip install tqdm")
-    os.system("pip install backports.weakref==1.0.post1")
-    os.system("pip install ChatterBot==0.7.6")
-    os.system("pip install enum34==1.1.6")
-    os.system("pip install funcsigs==1.0.2")
-    os.system("pip install gensim==3.1.0")
-    os.system("pip install jedi==0.11.0")
-    os.system("pip install libarchive==0.4.4")
-    os.system("pip install mock==2.0.0")
-    os.system("pip install parso==0.1.0")
-    os.system("pip install pbr==3.1.1")
-    os.system("pip install regex==2017.11.9")
-
+    download_github_code("common/requirements_colab.txt")
```

Comment:
Any reason we're not using the [requirements.txt from Docker](https://github.com/hse-aml/natural-language-processing/blob/master/docker/requirements.txt) here?

Reply:
We don't need most of these packages in Colab (e.g, all jupyter* packages, Werkzeug, widgetsnbextension) -> we can save time not installing them.  Also, some of them might be conflicting with the Colab environment (but I can't tell this for sure now).